winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+`ExportToExcel` method.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.SelectedItems, options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-selected-item.png" alt="Exporting Selected Items only to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Row Height and Column Width customization 
+
+After exporting data to excel, you can set different row height and column width for the columns based on your requirement. You can refer [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation#adjust-row-height-and-column-width) for more information. 
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].SetRowHeight(2, 50);
+workBook.Worksheets[0].SetColumnWidth(2, 50);
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+## Styling cells based on CellType in Excel
+
+You can customize the cell styles based on `CellType` by using `GridExportHandler`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.GridExportHandler = GridExportHandler;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private static void GridExportHandler(object sender, DataGridExcelExportStartOptions e)
+{
+    if (e.CellType == ExportCellType.HeaderCell)
+    {
+        e.Style.Color = Color.FromArgb(100, 228, 234);
+        e.Style.Font.Color = ExcelKnownColors.White;
+        e.Handled = true;
+    }
+
+    else if (e.CellType == ExportCellType.RecordCell)
+    {
+        e.Style.Color = Color.FromArgb(240, 224, 144);
+        e.Handled = true;
+    }
+
+    else if (e.CellType == ExportCellType.GroupCaptionCell)
+    {
+        e.Style.Color = Color.FromArgb(252, 159, 161);
+        e.Handled = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-cell-style-customization.png" alt="WinUI DataGrid displays Customized Cell Style based on CellType in Exported Excel" width="100%" Height="Auto"/>
+
+## Cell customization in Excel while exporting
+
+You can customize the cells by setting `CellsExportHandler` in `DataGridExcelExportOptions`.
+
+### Customize cell value while exporting
+
+You can customize the call values while exporting to excel by using `CellsExportHandler` in `DataGridExcelExportOptions`.
```

Comment:
Customize the cell values while exporting to excel by using the `CellsExportHandler` in the `DataGridExcelExportOptions.`

Reply:
Content modified